results/intfloat__multilingual-e5-small/MLSUMClusteringP2P.json

```diff
@@ -0,0 +1,147 @@
+{
+  "dataset_revision": "b5d54f8f3b61ae17845046286940f03c6bc79bc7",
+  "mteb_dataset_name": "MLSUMClusteringP2P",
+  "mteb_version": "1.7.26",
+  "test": {
+    "de": {
```

Comment:
You may want to check this, the language code is different in the results because it uses the language of the HF subset. 
I suppose it'll be handled later with older results?